libs/ui/src/screens/activity/components/activity-item.tsx

```diff
@@ -108,3 +102,5 @@ export const ActivityItem: FC<Props> = ({
     </Column>
   );
 };
+
+export default memo(ActivityItem);
```

Comment:
can we avoid default export? 
it's not typical for our project

Reply:
maybe..